lib/modules/manager/gomod/artifacts.ts

```diff
@@ -271,7 +273,16 @@ export async function updateArtifacts({
 
     const execCommands: string[] = [];
 
-    let args = 'get -d -t ./...';
+    const userGoGetDirs = config.goGetDirs?.filter(isValidLocalPath);
+    if (config.goGetDirs?.length !== userGoGetDirs?.length) {
+      logger.debug(`some of the directories specified in "userGoGetDirs" are invalid, those will be skipped.
+      configured: ${config.goGetDirs?.join(', ') ?? ''}
+      will be used: ${userGoGetDirs?.join(', ') ?? ''}
```

Comment:
@rarkins i think we should log that as metadata?

Reply:
what do you mean by "log as metadata"?

Reply:
like `logger.debug({ userGoGetDirs }, 'Blah blah')`